test-harness/src/test/java/com/netflix/conductor/tests/integration/WorkflowLegacyMigrationTest.java

```diff
@@ -118,4 +118,10 @@ private String getWorkflowResourcePath(String workflowName) {
      */
     public void testForkJoinNestedWithSubWorkflow() {
     }
+
+    @Ignore
+    @Test
+    @Override
+    public void testExecutionTimes(){}
```

Comment:
Why is this test ignored?

Reply:
I tought that these tests were not supposed to run based on new code. These legacy tests are a bit confusing and to be honest I still don't get how it works properly.

Can you explain why these legacy tests exists and how to test it properly? Even making a test that passes, I don't know if I mounted the .json correctly.

Reply:
The legacy tests were added when moving to Conductor 2.0 to ensure backwards compatibility such that workflows in progress are not affected by the rolling update. As long as the test is added to the `AbstractWorkflowServiceTest` class, it will be run as both set of tests (legacy and current).